src/specs/RNGestureHandlerButtonNativeComponent.ts

```diff
@@ -11,7 +11,7 @@ interface NativeProps extends ViewProps {
   foreground?: boolean;
   borderless?: boolean;
   enabled?: WithDefault<boolean, true>;
-  rippleColor?: ColorValue;
```

Comment:
Is it possible to avoid overwriting this type?
Color may also be described as a string (`red` or `green`), replacing this type with `Int32` removes that possibility.